ui/packages/shared/profile/src/MatchersInput/index.tsx

```diff
@@ -79,7 +79,12 @@ const MatchersInput = ({
 
       call.response
         .then(response => {
-          setLabelValues(response.labelValues);
+          // replace single `\` in the `labelValues` string with doubles `\\` if available.
+          const newValues = response.labelValues.map(value =>
+            value.includes('\\') ? value.replace('\\', '\\\\') : value
```

Comment:
I belive the `string.replace` will only replace the first matching pattern, so it wouldn't handle a second occurrence of `\\` in the string. 
I am not sure if multiple escape characters is a possibility here, but just in case.

Reply:
Also, Should we extract this into a `sanitizeLabelValue` function in `@parca/functions` package and use it?

Reply:
Yeah I just figured that was a possibility. making a change now

Reply:
It's great that the CodeQL scanner captured it too. 😀